src/apps/Agreement/AgreementHeader.js

```diff
@@ -0,0 +1,200 @@
+import React, { useMemo, useCallback } from 'react'
+import {
+  IconGrid,
+  textStyle,
+  GU,
+  DropDown,
+  Tag,
+  useTheme,
+  IconCheck,
+  useLayout,
+  IconWrite,
+  IconShare,
+} from '@aragon/ui'
+import PropTypes from 'prop-types'
+import { STATUS_PENDING, STATUS_ACTIVE } from './agreement-statuses'
+import icon from './assets/icon.svg'
+
+function AgreementHeader({ title, status, onSign, onShare }) {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+
+  const compactMode = layoutName === 'small'
+
+  const handleDropdownChange = useCallback(
+    index => {
+      if (index === 0) {
+        onSign()
+      }
+      if (index === 1) {
+        onShare()
+      }
+    },
+    [onSign, onShare]
+  )
+
+  const dropdownItems = useMemo(
+    () => [
+      <DropdownItem Icon={IconWrite} label="Sign" />,
+      <DropdownItem Icon={IconShare} label="Share" />,
+    ],
+    []
+  )
+
+  return (
+    <div
+      css={`
+        display: flex;
+        align-items: center;
+        margin-bottom: ${compactMode ? 4 * GU : 5 * GU}px;
+      `}
+    >
+      {compactMode && <DecorativeIcon />}
+      <div
+        css={`
+          display: flex;
+          flex: 1;
+          justify-content: space-between;
+        `}
+      >
+        <div>
+          <h1
+            css={`
+              ${compactMode ? textStyle('title3') : textStyle('title2')};
+              line-height: 1.3;
```

Comment:
Oh interesting, we usually set the `line-height` through the `textStyle()`.

Does it look weird without this override?